Section3/Exercise1/Main.cpp

```diff
@@ -9,7 +9,8 @@ int main() {
 	cout << " Please enter a number between 1-10 please" << endl;
 	cin >> Number;
 	int Counter = 0;
-	while (Number > 10 || Number < 1 || (!(cin >> Number))) 
+
+	while (cin.good()==false || Number > 10 || Number < 1 ) 
```

Comment:
This is good, but keep in mind that it's more common to check `!cin.good()` rather than `cin.good() == false`, especially since the other way around would be `cin.good() == true` instead of `cin.good()`.